src/common/filters/global-exception.filter.spec.ts

```
@@ -0,0 +1,169 @@
+        getResponse: mockGetResponse,
+        getRequest: mockGetRequest,
+      }),
+      getArgByIndex: jest.fn(),
+      getArgs: jest.fn(),
+      getType: jest.fn(),
+      switchToRpc: jest.fn(),
+      switchToWs: jest.fn(),
+    } as unknown as ArgumentsHost;
+
+    // Reset NODE_ENV for consistent testing of 'details' field
+    delete process.env.NODE_ENV;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)